src/replicache.test.ts

```diff
@@ -24,6 +24,7 @@ import {WriteTransactionImpl} from './transactions';
 import {emptyHash, Hash} from './hash';
 import {defaultPuller} from './puller';
 import {defaultPusher} from './pusher';
+import * as licensing from '@rocicorp/licensing/src/client';
```

Comment:
It's more conventional to directly import the names used, like:

```
import { PROD_LICENSE_SERVER_URL, TEST_LICENSE_KEY, LicenseStatus, getLicenseStatus} from ...
```

In the case where the imported names are overly generic or conflict with some other name, they can be renamed like:

```
import {foo as bar} from ....
```

Reply:
In a file with lot of imports I *like* having the name of the module prefix the thing I'm using, like `licensing.foo` is a good reminder where foo comes from and what it is. Is that not something people do? By your comment guessing no...

Reply:
Yeah it's not commonly done. I guess because the ability to rename identifiers at import time addresses the concern of possibly ambiguous names without imposing the cost of `licensing.` on names that are already unambiguous.